WordPress/src/main/java/org/wordpress/android/ui/prefs/AppPrefs.java

```diff
@@ -113,7 +113,9 @@
         NEWS_CARD_DISMISSED_VERSION,
         // Store a version of the last shown News Card
         NEWS_CARD_SHOWN_VERSION,
-        AVATAR_VERSION
+        AVATAR_VERSION,
+        //Gutenberg editor enabled
```

Comment:
Hmm, this comment seems to just be repeating the name of the constant below it so, I'd say let remove it or expand it if it has to convey something more than what is understood by the constant name. WDYT?

Reply:
Fixed. (removed)